server/services/game_service.go

```diff
@@ -59,13 +63,32 @@ func (g *GameServiceImpl) CreateGame(lobby Lobby) *GameInstance {
 		Rules:             rules.DefaultGameRules,
 		InputRules:        rules.DefaultInputRules,
 		ActiveConnections: lobby.ActiveConnections,
-
 		PlayerSubmissions: make(map[string]pogo.TurnSubmissionMsg),
+		Log:               getGameLogger(lobby.ID),
 	}
 	g.activeGames[newInstance.ID] = newInstance
+	fmt.Println("Game created: ", newInstance.ID)
+	newInstance.Log.Info("Game created")
 	return newInstance
 }
 
+func getGameLogger(gameID string) *logrus.Logger {
+	f, err := files.GetLogFile(gameID)
```

Comment:
Ok, so each time a new game instance is created, it carries around a log file that has the game code and creation timestamp as its name.  Correct?

Reply:
Yessir, that's the plan. I don't have any log cleanup right now, but we also aren't going to be playing a ton of games yet -- so I'll cross that bridge when we get there.